src/main/scala/com/evolutiongaming/cluster/ActorRefExtension.scala

```diff
@@ -0,0 +1,5 @@
+package com.evolutiongaming.cluster
+
+import akka.actor.{ActorRef, Extension}
+
+class ActorRefExtension(val ref: ActorRef) extends Extension
```

Comment:
```object MappedAllocationStrategyDDProxy extends ExtensionId[ActorRefExtension] {
  override def createExtension(system: ExtendedActorSystem): ActorRefExtension =
    new ActorRefExtension(system actorOf Props[MappedAllocationStrategyDDProxy])
```
The problem is that `createExtension` should return an `Extension`, it can't return just an `ActorRef` and in this case the `Extension` is `ActorRefExtension`